Woof/Woof/Model/Walking/WalkingStatus.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+
+/**
+ The possible walking status values for a `Walking`.
+ Conforms to the `Codable` protocol.
+ */
+enum WalkingStatus: Codable {
+    /// The walking is available for booking.
+    case available
+
+    /// The walking is pending approval from the pet sitter.
+    case pending
+
+    /// The walking is booked by owner.
+    case booked
+
+    /// The walking has been completed.
+    case finished
+
+    /// The walking has been canceled by the sitter.
+    case canceledBySitter
+
+    /// The walking has expired due to the elapsed duration since it was initially declared.
+    case expired
+
+    /// Text description of possible status.
+    var description: String {
```

Comment:
Why is this property needed? Maybe make the _enum_ with _String_ associated type: 
`enum WalkingStatus: String, Codable { ..}` 
and use _rawValue_?

Reply:
Yes, the associated type it's a more lightweight variant.
Fixed with 7f70e76